Home page

prathamdedhiya17 opened the pull request:
![Desktop-01](https://user-images.githubusercontent.com/68095465/215163762-6833cd71-2e47-4076-b1f7-71d9639500dc.png)
![Desktop-02](https://user-images.githubusercontent.com/68095465/215163772-4d6a1b68-e057-4100-8d36-8c5cc0f8578f.png)
![Desktop-03](https://user-images.githubusercontent.com/68095465/215163778-39ab346f-7002-42b4-97e1-7523e0b1dca5.png)
![Desktop-04](https://user-images.githubusercontent.com/68095465/215163785-976fda09-9363-4674-ad18-81e08e1f9fc2.png)
![Desktop-05](https://user-images.githubusercontent.com/68095465/215163789-2819335d-9e3a-490f-8c9e-dcb5c879d0d7.png)
![Desktop-06](https://user-images.githubusercontent.com/68095465/215163796-ddc33f04-745e-4d74-9796-4051368b8617.png)
![Mobile-01](https://user-images.githubusercontent.com/68095465/215163797-641b00c6-a59d-4d8b-afb5-6009c2a167ab.png)
![Mobile-02](https://user-images.githubusercontent.com/68095465/215163799-03a7a34b-45a1-45e2-a51d-0f91e3e4f5e1.png)
![Mobile-03](https://user-images.githubusercontent.com/68095465/215163805-545860e6-b69c-4003-8c27-d5a657a967b6.png)
![Mobile-04](https://user-images.githubusercontent.com/68095465/215163808-ea23a960-e232-4d63-ae31-08d79264137e.png)
![Mobile-05](https://user-images.githubusercontent.com/68095465/215163810-6bb17314-7ad3-4a80-a418-3dc4095a43fa.png)
![Mobile-06](https://user-images.githubusercontent.com/68095465/215163814-fe9baa4c-5e45-4830-84fb-da2fa5904d15.png)
![Mobile-07](https://user-images.githubusercontent.com/68095465/215163817-81919c34-852a-4220-aeaa-aee312de40f8.png)
![Mobile-08](https://user-images.githubusercontent.com/68095465/215163819-5ebdb105-97e8-4e26-b3bc-1d049115326a.png)
![Mobile-09](https://user-images.githubusercontent.com/68095465/215163822-c47ef2fa-e2dd-4d6e-849b-37ec5d18ea21.png)
![Tablet-01](https://user-images.githubusercontent.com/68095465/215163824-72ea5036-d770-4ff9-bbf9-c7690b837397.png)
![Tablet-02](https://user-images.githubusercontent.com/68095465/215163825-4fd286f6-0fe4-4992-98f7-3f4cf7bdb383.png)
![Tablet-03](https://user-images.githubusercontent.com/68095465/215163827-e46947c1-0bd6-461d-ad99-15afe42b93f6.png)
![Tablet-04](https://user-images.githubusercontent.com/68095465/215163831-c19b11e8-4f56-48aa-8beb-363be9c53427.png)
![Tablet-05](https://user-images.githubusercontent.com/68095465/215163832-5bd6fba4-78cd-48b9-a641-7e85514b173e.png)

prathamdedhiya17 commented:
![Desktop-07](https://user-images.githubusercontent.com/68095465/215239404-e3f29749-94b3-4ff7-90b5-e49f95601f51.png)
![Mobile-10](https://user-images.githubusercontent.com/68095465/215239408-aad55f13-7b48-4483-b981-9a048f911c3a.png)
![Tablet-06](https://user-images.githubusercontent.com/68095465/215239410-9a1ad7b9-6bbe-46bc-949e-e741791450f0.png)

parnavh commented:
## Changes to be implemented:

- [ ] Use yarn instead of npm
- [ ] Do not reupload assets
- [ ] Do not make breaking changes to files eg. `Layout.astro`
- [ ] Improve UI (Improve position of assets)
- [ ] Use folder for your specific components

prathamdedhiya17 commented:
Implemented all the required changes and added a temporary fix to the event card.

![image](https://user-images.githubusercontent.com/68095465/221370373-ee2901f4-385e-4289-bdf9-9ea6ed99a984.png)
